be/src/olap/olap_index.h

```diff
@@ -87,13 +83,11 @@ struct OLAPIndexOffset {
 
 // 唯一标识一个RowBlock在Data文件和Index文件中的位置
 struct RowBlockPosition {
```

Comment:
It looks like `RowBlockPosition` and `OLAPIndexOffset` are the same thing, both use  (segNum, blockNum) pair to unique identify each row block within a segment group. Can we combine them into one?

Reply:
We should do this after be_refactor has been merged. And I think we should do it in anther patch.

Reply:
Ok